thirdparty/libssh2/include/libssh2_publickey.h

```diff
@@ -0,0 +1,119 @@
+/* Copyright (c) 2004-2006, Sara Golemon <sarag@libssh2.org>
```

Comment:
Is this patch from their code base or from elsewhere? If the latter, then it needs to be documented accordingly, otherwise it's covered by the LICENSE block....though I suspect they're actually from a different version?

Reply:
The include directory is copied verbatim from the same version we use in the ExternalProject. I've seen this done with all of our thirdparty dependencies, and I suspected that this is because we don't want to depend on includes only available in the build directory after building the external project. If this is not the case, I will revise this.
The patch file and the copied CMakeLists.txt, which are applied in the PATCH phase are my own modifications (on Windows we don't have a patch command, so the modified CMakeLists.txt is copied over the one in the downloaded sources).